notebooks/reinforcement_learning/solutions/contextual_bandits_with_tf_agents.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>nit: since we haven't use this often in other labs, can we link it to the documentation https://www.tensorflow.org/io/api_docs/python/tfio/bigquery/BigQueryReadSession#parallel_read_rows</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/160/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>

Reply:
<p>Done!</p>